Makefile

```diff
@@ -22,8 +22,7 @@ scrub: FORCE
 	find tutorial -name "*.ipynb" | xargs python tutorial/source/cleannb.py
 
 format: FORCE
-	yapf -i *.py pyro/distributions/*.py profiler/*.py docs/source/conf.py
-	isort --recursive *.py pyro/ tests/ profiler/*.py docs/source/conf.py
+	isort --recursive *.py pyro/ examples/ tests/ profiler/*.py docs/source/conf.py
```

Comment:
These are the only two lines I modified by hand. The rest of this diff was automatic.

Reply:
luckilyy for me those are the only two lines i read